message/validation/errors.go

```diff
@@ -56,6 +56,7 @@ var (
 	ErrEarlyMessage                        = Error{text: "early message"}
 	ErrLateMessage                         = Error{text: "late message"}
 	ErrTooManySameTypeMessagesPerRound     = Error{text: "too many messages of same type per round"}
+	ErrDuplicatedProposalWithDifferentData = Error{text: "duplicated proposal with different data"}
```

Comment:
I'd like @MatheusFranco99 's input here, did duplicate proposals just became ok?

Reply:
From my perspective, this type of error is clearly malicious and it should stay as rejected for releases.
If we're suspicious that our attacks is causing performance issue due to this rule, maybe we can test the attack without it in a deployment.
If it's really the issue, I think we should adapt/add some message validation rules to prevent against this specific attack, instead of ignoring duplicated proposals.
Maybe I'm missing something because I don't understand exactly how the attack triggers it.

Reply:
From a clear theoretical perspective @MatheusFranco99 is right.

However, currently we have a bug and on at least one instance at least we saw this being triggered wildly somehow. I found it hard to replicate.
However, if this issue gets triggered on mainnet somehow it will pose an issue.

Changing this to `ignore` won't open up any attack vectors and is the safe way to go.